Resources/ServerInfo/Rules.txt

```diff
@@ -57,18 +62,19 @@
 	• Fights and confrontations is completely disallowed on the safezone such as shooting, attacking or any forms of harm.
 	• Pirate actions may not begin on the Frontier. This includes stowing away while at the Frontier with intentions to steal the ship, outright stealing the ship or its cargo from the dock, selling someone else’s ship you hadn’t pirated outside the safe zone, or joining the crew under false pretenses with the intention to mutiny and take the ship.
 	• Dangerous research like artifacts and anomalies must not be done while inside the safezone boundaries.
-	• Do not transport any kind of dangerous creature inside the station.
+	• Do not transport any kind of creature, contraband or objects that might cause harm/death or bait into a conflict inside of the frontier station.
 
 [color=#a4885c][head=3]9. Security Forces and Station Staff roles are held to a higher standard of roleplay and are strictly non-antagonists.[/head][/color]
 	• Security Forces roleplaying as ‘bad cops’ and intentionally violating space law is forbidden. Space Law is not optional for these roles, and strong knowledge is required before taking a security position.
+	• Civilian and Security staff are expected to maintain a minimum of competency and performance on their jobs, following the Standard Operating Procedure as required.
```

Comment:
Is this intended to be future-proof for a station staff SOP, or is this intended to only cover the NFSD?  Only problem I could see with this is the lawyer is technically under the civilian department.  A reference to where the SOP is might be useful, too.
```suggestion
	• Civilian and Security staff are expected to maintain a minimum of competency and performance on their jobs. Security staff must follow NFSD Standard Operating Procedure (see Wiki in the menu for more details).
```